vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -289,6 +289,72 @@ void vtkF3DImguiActor::ReleaseGraphicsResources(vtkWindow* w)
 //----------------------------------------------------------------------------
 vtkF3DImguiActor::~vtkF3DImguiActor() = default;
 
+//----------------------------------------------------------------------------
+void vtkF3DImguiActor::RenderDropZone()
+{
+  if (this->DropZoneVisible)
+  {
+    ImGuiViewport* viewport = ImGui::GetMainViewport();
+    constexpr ImU32 color = IM_COL32(255, 255, 255, 255);
+
+    const int dropzonePad =
+      static_cast<int>(std::min(viewport->WorkSize.x, viewport->WorkSize.y) * 0.1);
+    const int dropZoneW = viewport->WorkSize.x - dropzonePad * 2;
+    const int dropZoneH = viewport->WorkSize.y - dropzonePad * 2;
+
+    constexpr float tickThickness = 3.0f;
+    constexpr float tickLength = 10.0f;
+
+    const int tickNumberW = static_cast<int>(std::ceil(dropZoneW / (tickLength * 2.0f)));
+    const int tickNumberH = static_cast<int>(std::ceil(dropZoneH / (tickLength * 2.0f)));
+
+    const double tickSpaceW =
+      static_cast<double>(dropZoneW - tickNumberW * tickLength) / (tickNumberW - 1);
+    const double tickSpaceH =
+      static_cast<double>(dropZoneH - tickNumberH * tickLength) / (tickNumberH - 1);
+
+    ::SetupNextWindow(ImVec2(0, 0), viewport->WorkSize);
+    ImGui::SetNextWindowBgAlpha(0.f);
+
+    ImGuiWindowFlags flags = ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoSavedSettings |
+      ImGuiWindowFlags_NoFocusOnAppearing | ImGuiWindowFlags_NoNav | ImGuiWindowFlags_NoMove;
+
+    ImGui::Begin("DropZoneText", nullptr, flags);
+    ImDrawList* draw_list = ImGui::GetWindowDrawList();
+
+    const ImVec2 p0(dropzonePad, dropzonePad);
+    const ImVec2 p1(dropzonePad + dropZoneW, dropzonePad + dropZoneH);
+
+    // Draw top and bottom line
+    for (float x = p0.x; x < p1.x; x += tickLength + tickSpaceW)
+    {
+      const float y0 = p0.y + tickThickness / 2.f;
+      const float x1 = std::min(p1.x, x + tickLength);
+      draw_list->AddLine(ImVec2(x, y0), ImVec2(x1, y0), color, tickThickness);
+      draw_list->AddLine(ImVec2(x, p1.y), ImVec2(x1, p1.y), color, tickThickness);
```

Comment:
I think this could be the reason of the rendering mismatch depending on the OpenGL implementation.  
`tickThickness / 2.f` is `1.5`, so the rasterizer might choose between 1 or 2 additional pixels.  
Let's try to be pixel-perfect (keep integer coordinates)
Here's a suggestion:

```suggestion
      const float x1 = std::min(p1.x, x + tickLength);
      draw_list->AddRectFilled(ImVec2(x, p0.y - 1.f), ImVec2(x1, p0.y + 1.f), color);
      draw_list->AddRectFilled(ImVec2(x, p1.y - 1.f), ImVec2(x1, p1.y + 1.f), color);
```

Maybe make sure that `x` is pixel-perfect too, `tickSpaceW` may need to be rounded.